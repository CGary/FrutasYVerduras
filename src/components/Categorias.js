import React from "react";
import { Link } from "react-router-dom";

export const Categorias = () => {
  return (
    <Main>
      <Container>
        <Categ>Categor√≠as</Categ>
        <Link to="/categorias" className="categorias-row a-button">
          Frutas
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Verduras
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Legumbres
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Hortalizas
        </Link>
      </Container>
    </Main>
  );
};

import styled from "styled-components";

const Main = styled.div`
  background-color: white;
  padding: var(--vl-borde-ext);
`;

const Container = styled.div`
  border-radius: var(--vl-border-radius);
  border: solid 1px var(--cr-borde);
  display: grid;
  grid-template-rows: repeat(5, 30px);
  .categorias-row {
    margin-left: var(--vl-borde-caja);
    margin-right: var(--vl-borde-caja);
    display: flex;
    align-items: center;
    font-size: var(--normal);
    letter-spacing: var(--vl-space-normal);
    border-bottom: solid 1px var(--cr-borde);
  }
  a:last-child {
    border: none;
  }
`;

const Categ = styled.div`
  font-size: var(--title2);
  letter-spacing: var(--vl-space-normal);
  color: var(--cr-negro);
  background-color: var(--cr-fondo);
  height: 30px;
  display: flex;
  align-items: center;
  padding-left: var(--vl-borde-caja);
  border-bottom: solid 1px var(--cr-borde);
  border-top-left-radius: var(--vl-borde-caja);
  border-top-right-radius: var(--vl-borde-caja);
`;
