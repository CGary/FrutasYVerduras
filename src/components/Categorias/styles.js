import styled from "styled-components";

export const Main = styled.div`
  background-color: white;
  padding: var(--vl-borde-ext);
`;

export const Container = styled.div`
  border-radius: var(--vl-border-radius);
  border: solid 1px var(--cr-borde);
  display: grid;
  grid-template-rows: repeat(5, 30px);
  .categorias-row {
    margin-left: var(--vl-borde-caja);
    margin-right: var(--vl-borde-caja);
    display: flex;
    align-items: center;
    font-size: var(--normal);
    letter-spacing: var(--vl-space-normal);
    border-bottom: solid 1px var(--cr-borde);
  }
  a:last-child {
    border: none;
  }
`;

export const Categ = styled.div`
  font-size: var(--title2);
  letter-spacing: var(--vl-space-normal);
  color: var(--cr-negro);
  background-color: var(--cr-fondo);
  height: 30px;
  display: flex;
  align-items: center;
  padding-left: var(--vl-borde-caja);
  border-bottom: solid 1px var(--cr-borde);
  border-top-left-radius: var(--vl-borde-caja);
  border-top-right-radius: var(--vl-borde-caja);
`;
