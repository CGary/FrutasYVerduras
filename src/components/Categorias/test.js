import React from "react";
// import { shallow } from "enzyme";
import { create } from "react-test-renderer";
import Categorias from ".";
import { BrowserRouter } from "react-router-dom";

// beforeEach(() => {
// });

describe("<Categoria />", () => {
  it("snapshot", () => {
    const wrapper = create(
      <BrowserRouter>
        <Categorias />
      </BrowserRouter>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
