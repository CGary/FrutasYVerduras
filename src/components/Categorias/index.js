import React from "react";
import { Link } from "react-router-dom";
import {Main, Container, Categ} from "./styles"

export default function Categorias() {
  return (
    <Main>
      <Container>
        <Categ>Categor√≠as</Categ>
        <Link to="/categorias" className="categorias-row a-button">
          Frutas
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Verduras
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Legumbres
        </Link>
        <Link to="/categorias" className="categorias-row a-button">
          Hortalizas
        </Link>
      </Container>
    </Main>
  );
}