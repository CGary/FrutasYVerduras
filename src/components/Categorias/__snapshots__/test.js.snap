// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Categoria /> snapshot 1`] = `
<div
  className="sc-bdnylx dTyNcz"
>
  <div
    className="sc-gtssRu efuwky"
  >
    <div
      className="sc-dlnjPT eddDRN"
    >
      Categor√≠as
    </div>
    <a
      className="categorias-row a-button"
      href="/categorias"
      onClick={[Function]}
    >
      Frutas
    </a>
    <a
      className="categorias-row a-button"
      href="/categorias"
      onClick={[Function]}
    >
      Verduras
    </a>
    <a
      className="categorias-row a-button"
      href="/categorias"
      onClick={[Function]}
    >
      Legumbres
    </a>
    <a
      className="categorias-row a-button"
      href="/categorias"
      onClick={[Function]}
    >
      Hortalizas
    </a>
  </div>
</div>
`;
